PendingReleaseNotes

```diff
@@ -57,6 +57,9 @@
   level storage operator (like Rook) to recreate the missing file system.
   See https://docs.ceph.com/en/latest/cephfs/administration/#file-systems
   docs for more information.
+* RADOS: A POOL_APP_NOT_ENABLED health warning will now be reported if
```

Comment:
maybe add a sentence to say what users need to do to resolve or mute the warning?